packages/shared/src/components/profile/devcard/DevCard.tsx

```diff
@@ -97,7 +102,7 @@ export function DevCard({
               style={{ borderColor: devcardBorder }}
             />
             <img
-              src={user.image}
+              src={user?.image ?? fallbackImages.avatar}
```

Comment:
This shouldnt be needed anymore right with above fix?

Reply:
No needed, but I think is just a double check just in case we cannot fetch user image in future. If we have other checks I can remove it!